plugin.json

```diff
@@ -35,7 +34,11 @@
       "method": "redirectLogout"
     }
   ],
-  "acpScripts": [
-    "static/lib/admin.js"
-  ]
+  "nbbpm": {
+    "compatibility": "^2.0.0"
```

Comment:
Doesn't this mean that it is only compatible with 2.0.0 and above? I tested and it seems to work with 1.18 too.